gpii/node_modules/solutionsRegistry/test/all-tests.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+"use strict";
+
+require("./GenericTermsTests");
+
```

Comment:
Blanks seem unnecessary - also, including the exact filename with extension is more in keeping with other rollups

Reply:
Done.